lib/task-data/tasks/create-raid.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015, EMC, Inc.
+/* jshint multistr:true */
+
+module.exports = {
+    friendlyName: 'Create RAID via Storcli',
+    injectableName: 'Task.Raid.Create',
```

Comment:
I suggest a name change, since this is specific to MegaRAID commands rather than a generic RAID creation task. Perhaps `Task.MegaRAID.Create` or `Task.Raid.Create.MegaRAID`?